alephmarcreader/abstractalephmarcreader.py

```diff
@@ -232,6 +257,27 @@ def get_mentioned_person(self):
         return mentioned
     get_mentioned_person.__annotations__ = {'return': [Person]}
 
+    def get_standardized_date(self):
+        """
+        Returns the standardized date.
+        :return: [StandardizedDate]
+        """
+        std_date = []
+        for field in self.__get_field('046'):
+            date_start = self._handle_subfields_cardinality_max_one(self.__get_subfield_texts(field, 'c'), '046', 'c')
+            date_end = self._handle_subfields_cardinality_max_one(self.__get_subfield_texts(field, 'e'), '046', 'e')
+
+            if date_end:
+                date_end = date_end.replace(".", "-")
+
+            if date_start:
+                date_start = date_start.replace(".", "-")
```

Comment:
see comment above

Reply:
see reply above

Reply:
see comment to your reply to my comment above